packages/runtime/container-runtime/src/connectionTelemetry.ts

```diff
@@ -361,17 +361,20 @@ class OpPerfTelemetry {
 			// The threshold could be adjusted, but ideally it stays  workload-agnostic, as service
 			// performance impacts all workloads relying on service.
 			const category = duration > latencyThreshold ? "error" : "performance";
-			this.opLatencyLogger.sendPerformanceEvent({
-				eventName: "OpRoundtripTime",
-				sequenceNumber,
-				referenceSequenceNumber: message.referenceSequenceNumber,
-				duration,
-				category,
-				pingLatency: this.pingLatency,
-				msnDistance:
-					this.deltaManager.lastSequenceNumber - this.deltaManager.minimumSequenceNumber,
-				...latencyData.opPerfData,
-			});
+			if (message.type !== "noop") {
```

Comment:
Great find. Before we merge this, we need to determine if there is a value to emitting the noop version of this message If there is, Maybe the change we make instead is to ignore it within devtools. And if not, perhaps we should include the message type here?

Reply:
Also need to confirm, it is a bug for noops to be missing statistics. If so maybe we can make a change here so that they do.